Client/src/Pages/Infrastructure/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Button } from "@mui/material";
+import { Heading } from "../../Components/Heading";
+
+function Infrastructure() {
```

Comment:
Please do add the JSdoc for this page, always nice to have even if there's nothing that needs config here.

Reply:
All right!

Reply:
Done here
https://github.com/bluewave-labs/bluewave-uptime/pull/1152/commits/bfef16fb2099583413c0c56ef6ee17c7e120b09e